### ISSUE TITLE
Question about proteomicslfq output format

### ISSUE THREAD

**Issue by mpage21** [bigbio/quantms#239]
I am using quantms and I would like some help understanding the output. Here is my original issue: 
https://github.com/nf-core/proteomicslfq/issues/192

**Comment by ypriverol** [comment]
Thanks @mpage21 for using quantms. Here some details about the mzTab: 

> Specifically, I am looking for information about the opt_ columns and how to interpret the values in them. 

In mztab optional columns are columns not mandatory in the main specification, meaning some pipelines will export them and some other pipelines no. For quantms, we try to be consistent and always for TMT and LFQ-DDA export the same optional columns, but it is not always possible. However, for the same combination of parameters and type of analysis, for sure 100% you will have the same set of optional columns. 

> For example, in the protein table, there is a column opt_global_result_type. The majority of the values in that column are either protein_details or single_protein, but I don't understand what the difference is. Could you explain or could point me to documentation that explains those opt_ columns specific to this pipeline's output?

For the `opt_gobal_result_type` you should always use the proteins that contain `single_protein` in the value. This column is used to represent the ambiguity proteins, if the protein is the `single_protein`, it means that protein is the main protein of the protein group and can be distinguished, the `protein_details` is used for all the proteins identified within a protein group and contains for example information of their coverage, number of unique peptides, intensities etc. 

@timosachsenberg @jpfeuffer Can also share more ideas about the concept.

**Comment by jpfeuffer** [comment]
Hi! Yes, for the result_type:

- `indistinguishable_protein_group`: a group of proteins for which all of the peptides matching its protein sequence that pass FDR filtering equally map to all of the members. Therefore they are in this experiment indistinguishable and are quantified identically.
- `protein_details`: Details about proteins in an indistinguishable group. Although proteins in a group are quantified together, they still have their own sequence (and therefore coverage) and descriptions. This cannot be represented in one row per group in mzTab. Therefore we add these auxiliary rows per member.
- `single_protein`: a protein with at least one unique/distinguishing identified peptide which can therefore be fully represented in a single row.

TLDR: Use indistinguishable_protein_group and single_protein for almost all of your downstream analysis, except if you need the exact sequence or coverage for each protein.

**Comment by mpage21** [comment]
Thank you so much for the explanation! What about the other optional columns? Is there some documentation that I can read about those?

**Comment by mpage21** [comment]
@ypriverol @jpfeuffer 

Do you know where I can find documentation on the optional columns that are used in this pipeline? I am still confused by many of them. Thank you!
